jenkins/tests/test_upgrade_python_requirements.py

```diff
@@ -46,9 +48,10 @@ def test_no_changes(self, delete_branch_mock, get_user_mock, create_branch_mock,
            return_value=[])
     @patch('jenkins.pull_request_creator.PullRequestCreator.github_helper.get_github_instance',
            return_value=Mock())
-    @patch('jenkins.pull_request_creator.PullRequestCreator.github_helper.connect_to_repo', return_value=Mock())
+    @patch('jenkins.pull_request_creator.PullRequestCreator.github_helper.repo_from_remote', return_value=Mock())
```

Comment:
Nit: If you wanted to clean this up, you might be able to just setup mock_github_helper, and then send that to a method that sets up all the other mocks to help clean this up.  I do not know if this would work, and it is certainly not required.